src/main/java/org/pircbotx/Configuration.java

```diff
@@ -245,23 +245,23 @@ protected Configuration(Builder builder) {
 		/**
 		 * Enable or disable sending WEBIRC line on connect, default disabled
```

Comment:
If you're going to remove the defaults and just keep it blank, the code documentation must reflect that.